tests/jballerina-integration-test/src/test/resources/configurables/configStructuredTypesProject/configStructuredTypes/modules/mod3/mod3.bal

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Shall we properly structure the test project and rename modules accordingly.

Reply:
We can also create new files within the modules so that each file represent a separate area that we are testing. If this is too much for this PR we can do it in another PR. If so let's create an issue.